client/src/pages/CreateNewPage.css

```diff
@@ -0,0 +1,46 @@
+.Form {
+}
+
+.Page__heading {
+  border: 2px none black;
+  padding: 2em 8em;
+  border-radius: 15px;
+  background: #f4b7b7;
+  display: flex;
+  align-items: center;
+  color: #d84064;
+  margin: 3em;
+  overflow-y: auto;
+}
+#name {
+  height: 25px;
+}
+
+#recipe {
+  width: 44vw;
+  height: 25vh;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2em;
+}
+
+button {
+  border: 2px none black;
+  padding: 2em 8em;
+  border-radius: 15px;
+  background: #f4cbad;
+  display: flex;
+  align-items: center;
+  color: #d84064;
+  margin: 3em;
+  overflow-y: auto;
+}
+
+input {
+  border-radius: 15px;
+  border: none;
```

Comment:
why do you define a border and write none again?

Reply:
```border: none```and ```border:hidden``` or ```border: 0``` call more or less the same feature - borders disappear. I guess I am more used to removing the border line with ```none```